src/L.Control.Heightgraph.js

```diff
@@ -183,17 +180,21 @@ import {
             }
         },
         /**
-         * Removes the drag rectangle and zooms back to the total extent of the data.
+         * Removes the drag rectangle
+         * @param {boolean} skipMapFitBounds - whether to zoom the map back to the total extent of the data 
          */
-        _resetDrag() {
+        _resetDrag(skipMapFitBounds) {
```

Comment:
I introduced the arg as is in order to keep the function backwards compatible; if the function is called with no arg (i.e. all existing calls to it), it's as if the caller want to fit the map to bounds (i.e. the previous behavior). The caller would have to explicitly pass `true` as arg to skip the fitBounds call from being made.